pkg/server/sandbox_remove.go

```diff
@@ -49,7 +50,10 @@ func (c *criService) RemovePodSandbox(ctx context.Context, r *runtime.RemovePodS
 	// Return error if sandbox container is still running or unknown.
```

Comment:
Can you add another commit to fix this comment?

Reply:
Sure, surprised this is still in master. 
https://github.com/containerd/cri/blob/a0b3b4e4da2e7802927f67efc77feb5433882b0e/pkg/server/sandbox_remove.go#L49

Reply:
Done